prometheus-ksonnet/lib/kausal.libsonnet

```diff
@@ -233,6 +233,21 @@ k {
       deployment.mixin.spec.template.spec.withVolumesMixin([
         volume.fromConfigMap(name, name),
       ]),
+      
+    configVolumeMountSubPath(name, path, subpath)::
+      local container = $.core.v1.container,
+            deployment = $.extensions.v1beta1.deployment,
+            volumeMount = $.core.v1.volumeMount,
+            volume = $.core.v1.volume,
+            addMount(c) = c + container.withVolumeMountsMixin(
+        volumeMount.new(name, path) +
+        volumeMount.withSubPath(subpath)
```

Comment:
You could add an optional param to `configVolumeMount(..., volumeMountMixin={})` and then pass `withSubPath` there.  Not sure thats better though.  WDYT?

Reply:
Sorry for delay!  If you have a minute, make it a optional parameter.  If not let me know.